meta-lmp-base/classes/lmp.bbclass

```diff
@@ -64,17 +75,17 @@ run_fiotool_cmd () {
 do_image_ostreepush[depends] += "ostreeuploader-native:do_populate_sysroot"
 IMAGE_CMD:ostreepush:prepend () {
 	if [ "${USE_FIOTOOLS}" = "1" ]; then
-		run_fiotool_cmd "fiopush"
-		# force return so garage-push called from meta-updater's IMAGE_CMD:ostreepush is not executed
+		run_fiotool_cmd "${FIO_PUSH_CMD}" "fiopush"
+		# force return so garage-push called from meta-updater's IMAGE_CMD_ostreepush is not executed
```

Comment:
No need to change this comment line, the correct syntax now (>= honister) is indeed `IMAGE_CMD:ostreepush`.

Reply:
wtf, I don't recall changing it :).

Reply:
done

Reply:
Probably because you changed this in hardknott and moved back the file to honister? But all good now :-)